custom/icds_reports/ucr/data_sources/person_cases_v2.json

```diff
@@ -2211,6 +2211,48 @@
       {
         "column_id": "count",
         "type": "count"
+      },
+      {
+        "column_id": "aadhar_date",
+        "type": "expression",
+        "is_nullable": true,
+        "is_primary_key": false,
+        "create_index": true,
+        "datatype": "date",
+        "expression": {
+          "type": "nested",
+          "argument_expression": {
+            "type": "reduce_items",
+            "aggregation_fn": "first_item",
+            "items_expression": {
+              "type": "filter_items",
+              "items_expression": {
+                "type": "icds_get_case_history_by_date",
```

Comment:
The add household / edit member forms are filled out infrequently (like once or twice per person), so don't expect that this will be too slow if we can filter forms by XMLNS